source/features/highlight-deleted-and-added-files-in-diffs.tsx

```diff
@@ -65,6 +69,7 @@ void features.add({
 		pageDetect.isPRFile404,
 		pageDetect.isPRCommit404
 	],
-	init: oneTime(init),
+	init,
+	deinit: () => observer.abort(),
```

Comment:
Needed since when you change from a commit page to the file changes tab, we need to reload the list.